lib/util.js

```diff
@@ -147,11 +152,54 @@ function styleText(format, text, { validateStream = true, stream = process.stdou
       validateOneOf(key, 'format', ObjectKeys(inspect.colors));
     }
     if (skipColorize) continue;
-    left += escapeStyleCode(formatCodes[0]);
-    right = `${escapeStyleCode(formatCodes[1])}${right}`;
+    codes.push(formatCodes);
```

Comment:
```suggestion
    codes[codes.length - 1] = formatCodes;
```
(or, ArrayPrototypePush)